dags/lib/config.py

```diff
@@ -109,7 +109,7 @@ class EtlConfig(Enum):
         '250310_A00516_0657_AH2CWVDMX2_germinal_CAP',
     ]
 elif env == Env.PROD:
-    fhir_image = 'ferlabcrsj/clin-fhir:d701565'
+    fhir_image = 'ferlabcrsj/clin-fhir:fb03159ce47d6e9f11ec34c6832d9213a62f89d4'
```

Comment:
This is the docker hash => https://hub.docker.com/r/ferlabcrsj/clin-fhir/tags
Should be `5caba97` (merge commit hash based)